resources/test-runner.mjs

```diff
@@ -0,0 +1,96 @@
+import { TEST_INVOKER_LOOKUP } from "./test-invoker.mjs";
+
+export class TestRunner {
+    constructor(suite, test, params, callback, page, frame) {
+        this._suite = suite;
```

Comment:
We should use real private instance variables.

Reply:
done